tests/main/document-interfaces-url/task.yaml

```diff
@@ -38,15 +38,12 @@ execute: |
     # The expiry date is <expiration year>/<expiration month>      
     
     declare -A exclusion_map
-    exclusion_map["intel-qat"]="2024/10"
-    exclusion_map["microceph"]="2024/10"
-    exclusion_map["microceph-support"]="2024/10"
-    exclusion_map["pkcs11"]="2024/10"
-    exclusion_map["registry"]="2024/11"
-    exclusion_map["snap-interfaces-requests-control"]="2024/11"
-    exclusion_map["system-packages-doc"]="2024/10"
-    exclusion_map["xilinx-dma"]="2024/10"
-    exclusion_map["nomad-support"]="2024/10"
+    exclusion_map["intel-qat"]="2024/12"
+    exclusion_map["microceph-support"]="2024/12"
+    exclusion_map["registry"]="2024/12"
```

Comment:
This will not be enough, the page isn't linked in the docs since it's currently experimental and will be for at least some months